src/storage/mod.rs

```diff
@@ -163,7 +169,8 @@ impl<'a> Storage<'a> {
                     .map(|file| (file_path, file))
             })
             .map(|(file_path, file)| -> Result<_, Error> {
-                let (content, alg) = compress(file)?;
+                let alg = DEFAULT_COMPRESSION;
+                let content = compress(file, alg)?;
```

Comment:
What's the purpose of setting a variable to `DEFAULT_COMPRESSION` instead of using it directly?

Reply:
I put `DEFAULT_COMPRESSION` in a variable as that makes changing the function to accept the algorithm as a parameter easier in the future. I'm not too attached to that though, I can surely change this.